xrefs.tsv

```diff
@@ -258,3 +258,5 @@ CONSO00352	umls	C0451306
 CONSO00353	inchi	InChI=1S/C12H18N2O4/c13-4-2-1-3-11(12(17)18)14-6-9(8-15)5-10(16)7-14/h5-7,11,15H,1-4,8,13H2,(H-,16,17,18)/t11-/m0/s1
 CONSO00353	smiles	NCCCC[C@@H](C(=O)O)[N+]1=CC(=CC(=C1)[O-])CO
 CONSO00360	ncit	C20613
+CONSO00361  mro MRO:0000750 http://purl.obolibrary.org/obo/MRO_0000750
```

Comment:
gotta get those tabs right

Reply:
also this isn't formatted correctly. Just put the identifier

Reply:
@cthoyt when I try to edit it, it hows me proper tab spaces.

Reply:
either don't use the in browser editor, or copy/paste the tab character from one of the other ones since the browser automatically adds spaces in the place of the tabs when you use the keyboard